src/main/java/uk/gov/justice/probation/courtcaseservice/service/model/Breach.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.probation.courtcaseservice.service.model;
+
+import io.swagger.annotations.ApiModel;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDate;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@ApiModel("Breach")
+@Getter
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Breach {
+    private String id;
```

Comment:
Why string here ? The community API has the nsi id as a long ? Also, lower down, the treatment of convictionId as a string is an overhead in the comparable implementation (although not part of this change)

Reply:
i chose String because that seems to be the representation for ids in other models e.g. `documentId` `convictionId`.

agree there is some overhead in converting convictionId for comparisons, but i didn't want to change the existing conviction model.

Reply:
OK - understand the reason not to change the conviction. documentId really is a string - it's a GUID type thing. I'd like to see us using a long unless there's a reason for string

Reply:
done